dashboard/test/controllers/reports_controller_test.rb

```diff
@@ -256,6 +256,8 @@ class ReportsControllerTest < ActionController::TestCase
 
   generate_admin_only_tests_for :admin_stats
 
+  generate_admin_only_tests_for :search_for_teachers
```

Comment:
Definitely, I'll add them when functionality complexity increases and stabilizes.